internal/api/impl/migrate/migrate.go

```diff
@@ -174,9 +191,6 @@ func getListOfConditions(schemasPath string, defaultValue string) (string, error
 
 	var listOfConditions strings.Builder
 
-	// first append an empty kind field in order to be able to append a default, fallback case for not covered plugins at the end of the conditions list (see below)
-	listOfConditions.WriteString("kind: _\n")
```

Comment:
because of to the new switch-like syntax ;)

basically I replaced
```go
kind: _
if type = a {
  kind: "A" 
}
if type = b {
  kind: "B"
}
if kind == _|_ {
  kind: "default" 
}
```

by
```go
plugin = [
  if type = a {
    kind: "A" 
  },
  if type = b {
    kind: "B"
  },
  {
    kind: "default" 
  }
][0]
```

Reply:
oh ok, nice !